lib/galaxy/tools/xsd/galaxy.xsd

```diff
@@ -1358,6 +1358,8 @@ provides a demonstration of using this tag.
 </test>
```

Comment:
Maybe you can also add the `<outputs>` section of the tool to the above snippet.

Reply:
good idea